frontend/src/scenes/pipeline/Destinations.tsx

```diff
@@ -85,7 +85,21 @@ export function DestinationsTable({ inOverview = false }: { inOverview?: boolean
                             if (destination.backend === 'plugin') {
                                 return <RenderApp plugin={destination.plugin} />
                             }
-                            return <RenderBatchExportIcon type={destination.service.type} />
+                            return (
```

Comment:
Not related to this PR but UX wise it feels nicer to my eye having the icon be far left rather than dangling in the middle here.

Reply:
I'm going to leave it there atm - we should discuss this in relation to other pages too, how we want the columns to be organized